frontend/src/components/Dashboard/Dashboard.tsx

```diff
@@ -59,6 +59,8 @@ const Dashboard = () => {
     surname: Userfront.user.data.surName,
   };
 
+  console.log(Userfront.user.data.givenName)
```

Comment:
Please remove/comment this out once you are done with it

Reply:
I have removed/commentted all unnecessary codes